### ISSUE TITLE
circleci/mongo:3.6-ram broken 

### ISSUE THREAD

**Issue by steffektif** [circleci/circleci-images#387]
Our nightly tests failed from one night to another with this errormessage: 
```
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] MongoDB starting : pid=7 port=27017 dbpath=/dev/shm/mongo 64-bit host=dc9585f6a3a1
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] db version v3.6.12
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] git version: c2b9acad0248ca06b14ef1640734b5d0595b55f1
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] modules: none
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] build environment:
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten]     distarch: x86_64
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true }, storage: { dbPath: "/dev/shm/mongo", journal: { enabled: false } } }
2019-04-10T06:02:53.614+0000 I STORAGE  [initandlisten] exception in initAndListen: NonExistentPath: Data directory /dev/shm/mongo not found., terminating
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] now exiting
2019-04-10T06:02:53.614+0000 I CONTROL  [initandlisten] shutting down with code:100
```
Using another image like mongo:3.6.0-ram fixed the problem for us. 
Here is a job that failed with that error: https://circleci.com/gh/signavio/workflow-server/28314

**Comment by edieval** [comment]
Hello, Same problem here with circleci/mongo:4-ram

**Comment by bqevin** [comment]
Same here with circleci/mongo:3.6-ram

**Comment by tooky** [comment]
This is fixed for circleci/mongo:4-ram for me after #388 was merged.
